### DISCUSSION TITLE
DAI 2023-2024 - My IntelliJ + Maven project - @GuggisbergSimon

### DISCUSSION THREAD

**Discussion by GuggisbergSimon** [heig-vd-dai-course/heig-vd-dai-course#52] [Show and tell]
https://github.com/GuggisbergSimon/DAI-04-intellij-maven

**Comment by ludelafo** [comment]
Thank you!

I see your README file is empty. You might want to fill it to explain the project, how to run it, etc.

You might have notice a warning when you package the JAR with Maven. We did update the course last week to include a second transformer to exclude problematic files. You can find the updated content here: https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/04-java-intellij-idea-and-maven/COURSE_MATERIAL.md#update-the-pomxml-file-to-include-the-dependencies-in-the-jar-file
